lib/workers/repository/update/pr/body/config-description.ts

```diff
@@ -62,3 +66,28 @@ function scheduleToString(
   }
   return scheduleString;
 }
+
+/**
+ * Return human-readable cron schedule summary if the schedule is a valid cron
+ * else return null
+ */
+function getReadableCronSchedule(scheduleText: string[]): string | null {
+  // assuming if one schedule is cron the others in the array will be cron too
+  try {
+    new CronPattern(scheduleText[0]); // validate cron
+    return scheduleText
+      .map(
+        (cron) =>
+          capitalize(
+            cronstrue
+              .toString(cron, {
+                throwExceptionOnParseError: false,
+              })
+              .replace('Every minute, ', ''),
+          ) + ` ( ${String(cron)} )`,
```

Comment:
```suggestion
          ) + ` (${cron})`,
```
this hides typescript errors and should be avoided. I would also drop the spaces before braces

Reply:
The `String` can be removed but the braces are there to prevent a bug that arises with some cron schedules. 
For example: `schedule: ['* 1 * * *', '* 2 * * *']`
If no space is added b/w the schedule and the braces thentwo `*` disappear [repro](https://github.com/Rahul-renovate-testing/repro-32810-old/pull/2). It seems the comma gets italicized. Adding a space prevents that.